appinventor/blocklyeditor/src/msg/zh_cn/_messages.js

```diff
@@ -422,15 +422,15 @@ Blockly.Msg.zh.switch_language_to_chinese_cn = {
     Blockly.Msg.LANG_MATH_IS_A_NUMBER_INPUT_NUM = '是否为数字?';
     Blockly.Msg.LANG_MATH_IS_A_NUMBER_TOOLTIP = '判断该对象是否为数字。';
 
-    // Blockly.Msg.LANG_MATH_IS_A_DECIMAL_HELPURL = 'http://appinventor.mit.edu/explore/ai2/support/blocks/math#isnumber';
+    // Blockly.Msg.LANG_MATH_IS_A_DECIMAL_HELPURL = '/reference/blocks/math#isnumber.html';
```

Comment:
`math.html#isnumber`

Also, I think we should be uncommenting these lines. Because they are commented out right now, the blocks take the English translations rather than the Chinese (ditto zh_tw version below). This can be done in a separate PR.